package.json

```diff
@@ -39,7 +39,6 @@
     "code": "^5.2.0",
     "depcheck": "^1.4.0",
     "husky": "^4.2.5",
-    "js-yaml": "^3.14.0",
```

Comment:
did you remove this manually? if so, please do it with npm uninstall or it will remain in package-lock files